core/source/org/libreoffice/ide/eclipse/core/export/AntScriptExportWizard.java

```diff
@@ -87,7 +87,7 @@ public void init(IWorkbench workbench, IStructuredSelection selection) {
                     if (res != null &&
                         res.getProject().hasNature(OOEclipsePlugin.UNO_NATURE_ID) &&
                         ProjectsManager.getProject(res.getProject().getName()).getLanguage() != null) {
-                        // TODO: We need to check if the project is configured correctly and notify
+                        // XXX: We need to check if the project is configured correctly and notify
```

Comment:
Why this change?

Reply:
To make my life easier because TODOs appear in Eclipse as errors...